datajunction-server/datajunction_server/sql/dag.py

```diff
@@ -98,7 +98,15 @@ def check_convergence(path1: List[str], path2: List[str]) -> bool:
         if partial1 == partial2:
             return True
 
-    return False
+    # TODO: Once we introduce dimension roles, we can remove this.  # pylint: disable=fixme
```

Comment:
Yeah... let's make a call on this soon.

Reply:
Yeah, this seems like an easy thing to add and it's really more for organizational purposes